__tests__/app.test.js

```diff
@@ -66,16 +65,38 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe(`Bad request`);
       });
   });
-});
-// Responds with:
-
-// an article object, which should have the following properties:
+  describe("PATCH /api/articles/:article_id", () => {
+    test("200: responds with updated article object", () => {
+      const updates = { inc_votes: 10 };
+      return request(app)
+        .patch("/api/articles/6")
+        .send(updates)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toEqual({
+            article_id: 6,
+            title: "A",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "Delicious tin of cat food",
+            created_at: "2020-10-18T01:00:00.000Z",
+            votes: 10,
+          });
+        });
+    });
 
-// author which is the username from the users table
-// title
-// article_id
-// body
-// topic
-// created_at
-// votes
-// Errors to Conside
+    test("status: 400 response with a bad request message when invalid property data inputted", () => {
+      const input = {
+        surname: "Lewangoalski",
+      };
+      return request(app)
+        .patch("/api/articles/6")
+        .send(input)
+        .expect(400)
+        .then((res) => {
+          expect(res.body).toMatchObject({ message: "Bad request" });
+        });
+    });
+  });
```

Comment:
Need more error testing here, what is the article id is invalid or not found?

Reply:
article_id actually is invalid here. hmm I shall still do another test. Thank you

Reply:
if the article is not found, but the ID still a number then you should have 404 not found,
then you should have another test for an invalid article id (wrong data type like a string) which should be 400 invalid id

Reply:
also think about what could happen if the update that you send is incorrect or has missing information

Reply:
here just testing for invalid inc votes input which is good, also test for the other errors as written above

Reply:
Ouch, OK I got it I will do. Thanks